gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-account/src/main/java/io/gravitee/am/gateway/handler/account/resources/account/util/AccountRoutes.java

```diff
@@ -15,14 +15,24 @@
  */
 package io.gravitee.am.gateway.handler.account.resources.account.util;
 
+/**
+ * @author Donald COURTNEY (donald.courtney at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+
 public enum AccountRoutes {
     INDEX("/"),
     STATIC_ASSETS("/assets/*"),
-    PROFILE("/api/profile"),
-    FACTORS("/api/factors"),
+        PROFILE("/api/profile"),
     ACTIVITIES("/api/activity"),
     CHANGE_PASSWORD("/api/changePassword"),
-    CHANGE_PASSWORD_REDIRECT("/forgotPassword");
+    CHANGE_PASSWORD_REDIRECT("/forgotPassword"),
+    FACTORS("/api/factors"),
+    FACTORS_CATALOG("/api/factors/catalog"),
+    FACTORS_BY_ID("/api/factors/:factorId"),
+    FACTORS_OTP_QR("/api/factors/qr"),
```

Comment:
For the qr method.
I wondering if it providing the factorId into the body is the way to go. 
Why not using `/api/factors/:factorId/qr` ? (not a requirement, just an open discussion)